guides/nestor/ISPsetup.md

```diff
@@ -0,0 +1,308 @@
+---
+Layout: post
+Title: ISP setup ubuntu
+Author: Nestor N Torres
+---
+
+### Contact
+- Slack: @nanjuan on wcscusf.slack.com
+- [LinkedIn](https://www.linkedin.com/in/nestor-n-torres-737172a5/)
+- Email: nestor@nntorres.com
+
+### Table
+1. [Prerequisites](#id-link-to-section)
+2. [Summary](#Summary)
+3. [First logical group of Page](#First)
+4. [Second logical group of Page](#Second)
+5. [Third Logical group of pare](#Third)
+
+## Prerequisites <a id="id-link-to-section"></a>
+1. If you have not setup your network adapter look here https://goo.gl/vs4cHC.
```

Comment:
I prefer you use the []() format. It makes the guide look clean. when referencing links.